client/src/components/pages/GameContainer.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react'
+import { Button, Container } from 'semantic-ui-react'
+
+const GameContainer = ({ pokemonList, shufflePokemonList }) => {
+
+    const [fourPokemon, setFourPokemon] = useState([]);
+    const [correctOption, setCorrectOption] = useState({});
+    const [answered, setAnswered] = useState(false);
+    const [styledImage, setStyledImage] = useState({ filter: "brightness(0)", });
+    const [correctUrl, setCorrectUrl] = useState("");
+
+    useEffect(() => {
+        setFourPokemon([...pokemonList].splice(0, 4));
+    }, [pokemonList]);
+
+    useEffect(() => {
+        setCorrectOption(fourPokemon[Math.floor(Math.random() * fourPokemon.length)]);
+    }, [fourPokemon])
+
+    useEffect(() => {
+        const pokeNumber = correctOption?.url ? getPokeNumber(correctOption.url) : 214;
+        getCorrectImage(pokeNumber);
+    }, [correctOption])
+
+    const getPokeNumber = (url) => {
+        let correctOptionNum = url;
+        const numberRegEx = /(\d+)\/$/;
+        return ((correctOptionNum.match(numberRegEx) || [])[1]);
+    }
+
+    const getCorrectImage = (number) => {
+        setCorrectUrl(`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${number}.png`);
+    }
+
+    const handleAnswerClick = (selectedName) => {
+        setAnswered(true);
+        setStyledImage({
+            filter: null,
+        })
+        if (correctOption.name === selectedName) {
+            console.log(" you guessed correct!")
+        } else {
+            console.log("wrong");
+        }
+    }
+
+    const handleShuffleClick = () => {
+        setAnswered(false);
+        setStyledImage({ filter: "brightness(0)" });
+        shufflePokemonList(pokemonList);
+    }
+
+    return (
+        <Container textAlign='center'
+            style={{
+                backgroundColor: "white",
+            }}>
+            <img src={correctUrl} style={styledImage} />
+            <p><Button onClick={handleShuffleClick}> Shuffle </Button></p>
+            {fourPokemon.length >= 4 && <div>
```

Comment:
I eventually want to see this changed to a loop that maps over `fourPokemon` and outputs a `Button` for each, but that is not a blocking change right now.